### PR TITLE
[No ticket] Scan docker image with Trivy

### PR DESCRIPTION
**Pull request by zbedo**
This PR introduces:

- Docker image security scan, if there are CRITICAL findings, job will fail.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
- [ ] I've updated Swagger to reflect any API changes

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
Once the codacy thing is resolved, 👍

File: build.sh [hunk 1]
```diff
@@ -90,6 +90,9 @@ function docker_cmd()
         echo "Building $DOCKERHUB_REGISTRY:$HASH_TAG"
         docker build -t $DOCKERHUB_REGISTRY:${HASH_TAG} --file Dockerfile .
 
+        echo "Scanning docker image..."
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL $DOCKERHUB_REGISTRY:${HASH_TAG}
```
Comment (rushtong) [inline build.sh]: I think this is also what codacy is complaining about:
```suggestion
        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "$DOCKERHUB_REGISTRY":${HASH_TAG}
```

Comment (zbedo) [reply build.sh]: Thank you, nice catch!

**Comment by rushtong** [comment]
I think it is also unhappy with `${HASH_TAG}`

> Here are two common cases where this warning seems unnecessary but may still be beneficial:
> 
> `cmd <<< $var         # Requires quoting on Bash 3 (but not 4+)`
> `: ${var=default}     # Should be quoted to avoid DoS when var='*/*/*/*/*/*'`

**Comment by zbedo** [comment]
@rushtong thank you!

**Comment by rushtong** [comment]
@adrazhi Can you 👍 this when you're able to? I'd like to move this through, thank you!

**Review by adrazhi — APPROVED** [review]

**Comment by rushtong** [comment]
@zbedo - Typically we have the PR author do the squash and merge. Let me know if you'd rather I do that.

**Comment by zbedo** [comment]
@rushtong  Merging is blocked, the base branch restricts merging to authorized users. Can you merge it?

**Comment by rushtong** [comment]
Ah, sorry, thanks - will do.

**Pull request merged**